cloudant-client/src/main/java/com/cloudant/client/api/views/SettableViewParameters.java

```diff
@@ -124,6 +171,23 @@
          */
         RB keys(K... keys);
 
+        /**
+         * <p>
+         * Determine whether the view should be returned from a "stable" set of shards.
+         * </p>
+         * <p>
+         * See:
+         * </p>
+         * <ul>
+         * <li>{@link SettableViewParameters#STABLE_FALSE}</li>
+         * <li>{@link SettableViewParameters#STABLE_TRUE}</li>
+         * </ul>
+         *
+         * @param stable string indicating stable view behaviour
+         * @return the builder to compose additional parameters or build the request
+         */
+        RB stable(String stable);
```

Comment:
The docs say `stable` is a `boolean` I think we should change the type here to match and remove the associated constants.

Reply:
Done in 1dceeae3f4050abe9c6909eb1cb4df9be959f62b